Library/Homebrew/vendor/bundle/ruby/2.6.0/gems/warning-1.1.0/lib/warning.rb

```diff
@@ -0,0 +1,248 @@
+require 'monitor'
+
+module Warning
+  module Processor
+    # Map of symbols to regexps for warning messages to ignore.
+    IGNORE_MAP = {
```

Comment:
Ooo, nice. I wonder if we can use this to get `-W0` removed entirely. Opened https://github.com/Homebrew/brew/issues/10593 as another issue to track this.